pareto/utilities/process_data.py

```diff
@@ -311,6 +311,10 @@ def check_required_data(df_sets, df_parameters, config, model_type="strategic"):
                 "SWDRiskFactors",
             ],
             Objectives.cost_surrogate: ["DesalinationSurrogate"],
+            Objectives.environmental: [
+                "AirEmissionCoefficients",
+                "TreatmentEmissionCoefficients",
```

Comment:
Won't we need other emission coefficients and installation coefficients?

Reply:
What this tab is doing is making sure that if the user selects `Objectives.environmental` for the objective function, then they also include the input tabs "AirEmissionCoefficients" and "TreatmentEmissionCoefficients". All of the data required which is specific to the environmental objective function should be passed in those two tabs.

Reply:
Oh ok. Thanks.